### DISCUSSION TITLE
Multiple Countries

### DISCUSSION THREAD

**Discussion by orionleo** [ReaVNaiL/New-Grad-2024#55] [Ideas]
It would be great if there would be role listings for different countries as well. I am myself from India and it would be a lot more helpful if there were open job listings for India as well.

**Comment by ujjwalm29** [comment]
I would suggest creating another repo for a different region(like India). Otherwise, the number of postings on this are going to be out of control. Maybe the README of this one can have a link to other repos, in case you want to divert people.

**Reply by rohan12k** [reply]
Yeah thats a good suggestion.

**Reply by yashwanthrk** [reply]
@ujjwalm29 That makes sense.

**Reply by samiranghosh04** [reply]
Would it be okay if I create a repo?
